build.gradle

```diff
@@ -1,15 +1,15 @@
 buildscript {
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.3.1'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
gradle'ı göndermene gerek yok, senin android studio'a uyması için değiştirdiysen.